pkg/endpoint/endpoint.go

```diff
@@ -870,6 +870,28 @@ func parseEndpoint(ctx context.Context, owner regeneration.Owner, bEp []byte) (*
 	return &ep, nil
 }
 
+// NewDatapathConfiguration return the default endpoint datapath configuration
+// based on whether per-endpoint routes are enabled.
+func NewDatapathConfiguration() models.EndpointDatapathConfiguration {
```

Comment:
nit: Is there a reason we don't just return a `*models.EndpointDatapathConfiguration`?

Reply:
`parseEndpoint` and `CreateHostEndpoint` use it by value. Should I still change and dereference for those callers? :-/

Reply:
Ah, I missed that! In that case let's leave it as is.